lib/rules/prop-name-casing.js

```diff
@@ -15,6 +15,7 @@ const allowedCaseOptions = ['camelCase', 'snake_case']
 /** @param {RuleContext} context */
 function create(context) {
   const options = context.options[0]
+  const ignores = context.options[1]?.ignores || []
```

Comment:
Please allow ignore patterns as well, similar to the `ignores` option in other rules:

https://github.com/vuejs/eslint-plugin-vue/blob/b1fa3b9a444131a99b4fe4d7e025734f424c5dcd/lib/rules/custom-event-name-casing.js#L114